example/package.json

```diff
@@ -17,7 +17,7 @@
     "dicom-parser": "file:../node_modules/dicom-parser",
     "hammerjs": "file:../node_modules/hammerjs",
     "react": "file:../node_modules/react",
-    "react-cornerstone-viewport-hooks": "file:..",
+    "react-cornerstone-viewport-hooks": "file:../dist/index.modern.js",
```

Comment:
here was to get reloading behaviour of webpack...

Reply:
ok got it